packages/modal/style.ts

```diff
@@ -43,9 +43,12 @@ export const scrim = css`
 export const modal = css`
   border-radius: ${borderRadiusDefault};
   background-color: ${themeBgPrimary};
+  display: flex;
+  flex-direction: column;
   left: 50%;
   max-height: calc(100vh - ${modalInset * 2}px);
   outline: none;
+  overflow: auto;
```

Comment:
This has nothing to do with this regression.

While testing this I found an edge case where if a user's viewport height is ~200px or less, they can't click footer buttons